autoPyTorch/data/tabular_target_validator.py

```diff
@@ -213,15 +224,13 @@ def _check_data(self, y: SupportedTargetTypes) -> None:
             y (SupportedTargetTypes):
                 A set of features whose dimensionality and data type is going to be checked
         """
-
         if not isinstance(y, (np.ndarray, pd.DataFrame,
                               List, pd.Series)) \
                 and not issparse(y):  # type: ignore[misc]
             raise ValueError("AutoPyTorch only supports Numpy arrays, Pandas DataFrames,"
                              " pd.Series, sparse data and Python Lists as targets, yet, "
-                             "the provided input is of type {}".format(
-                                 type(y)
-                             ))
+                             "the provided input is of type {}".format(type(y))
```

Comment:
Please refer to [this](https://github.com/automl/Auto-PyTorch/pull/434/files#r895949345) comment